src/extension-runners/chromium.js

```diff
@@ -179,12 +178,13 @@ export class ChromiumExtensionRunner {
 
       if (userDataDir && profileDirName) {
         // copy profile dir to this temp user data dir.
-        await fs.copy(
+        await fs.cp(
```

Comment:
Added to Node 16.7 exactly 3 years ago

https://nodejs.org/api/fs.html#fspromisescpsrc-dest-options